resources/assets/js/components/global/form/Submit.vue

```diff
@@ -37,6 +46,7 @@ export default {
 			return this.$store.getters[`form/${getter}`](this.parentName);
 		},
 		submitParent() {
```

Comment:
This name is a bit misleading now. How about this?
```suggestion
		onSubmitClicked() {
```